README.md

```diff
@@ -64,50 +64,46 @@ run()
 
 See the [toolkit documentation](https://github.com/actions/toolkit/blob/master/README.md#packages) for the various packages.
 
-## Publish to a distribution branch
+## Package for distribution
 
-Actions are run from GitHub repos.  We will create a releases branch and only checkin production modules (core in this case). 
+Actions are run from GitHub repos.  Packaging the action will create a self encapsulated distribution runnable from the GitHub graph.
 
 Comment out node_modules in .gitignore and create a releases/v1 branch
```

Comment:
Is this line still relevant?

Reply:
you are correct

Reply:
Actually the second sentence added here is still relevant right?  The action is self encapsulated and runs from the dist folder by cloning the repo.

Reply:
I updated it

Reply:
I think we need to remove this line:
Comment out node_modules in .gitignore and create a releases/v1 branch

Reply:
yeah, I did right before I merged.  Thanks